portal-front/messages/en.json

```diff
@@ -11,17 +11,17 @@
     "Enabled": "Enabled",
     "Published": "Published"
   },
-  "CountryComboBox": {
-    "Placeholder": "Choose a country"
-  },
   "CapabilityDescription": {
-    "Title": "Available capabilities",
-    "Description": "You can add specific capabilities to Filigran users",
     "Capabilities": {
       "ADMINISTRATE_ORGANIZATION": "Bypass inside of an organization",
       "MANAGE_ACCESS": "Ability to invite other users to this organization",
       "MANAGE_SUBSCRIPTION": "Ability to manage and subscribe the organization to services"
-    }
+    },
+    "Description": "You can add specific capabilities to Filigran users",
+    "Title": "Available capabilities"
+  },
+  "CountryComboBox": {
+    "Placeholder": "Choose a country"
```

Comment:
What did you use to order translations keys ? I think we should automate this to be sure everyone pushes an order json and prevent having reorder changes. 🙁